### PR TITLE
German and Croatian translations

### PR DESCRIPTION
**Pull request by milotype**
Update of German and release of Croatian translations

## PR Type
What kind of change does this PR introduce?
- Translation 

## Other information
Croatian is based on en_US versions.
German has some minor corrections/changes and is now also ordered as the en_US version. Some strings in the former version where missplaced.

### CODE REVIEW

**Comment by JasonStein** [comment]
@WalterWolf49  Can you help review as well? If you guys have questions, you can always take a look at how VSCode translated these similar strings.

File: src/Notepads/Strings/de-DE/Settings.resw [hunk 1]
```diff
@@ -186,11 +186,11 @@
     <comment>PersonalizationPage AccentColorSettings UseWindowsAccentColorToggleSwitch On display text.</comment>
   </data>
   <data name="PersonalizationPage_BackgroundTintOpacitySettings_Description.Text" xml:space="preserve">
-    <value>Acryl Hintergrundfarbton Transparenz. Please note that Acrylic effect will be disabled when battery saver mode is on or if you turn off transparency effects in Windows Settings.</value>
+    <value>Opazität der Acryl-Hintergrundfarbe. Der Acryleffekt wird deaktiviert wird, wenn der Batteriesparmodus eingeschaltet ist oder wenn die Transparenzeffekte in den Windows-Einstellungen ausschaltet sind.</value>
```
Comment (beerisgood) [inline src/Notepads/Strings/de-DE/Settings.resw:189]: > Der Acryleffekt wird deaktiviert wird,

ein "wird" zu viel.
Besser:
Der Acryleffekt wird deaktiviert,

Comment (JasonStein) [reply src/Notepads/Strings/de-DE/Settings.resw:189]: I will fix it.

Comment (JasonStein) [reply src/Notepads/Strings/de-DE/Settings.resw:189]: https://github.com/JasonStein/Notepads/commit/f886a1cfdc618dcc51f03cf4350e680aefc346ba

Comment (beerisgood) [reply src/Notepads/Strings/de-DE/Settings.resw:189]: > [f886a1c](https://github.com/JasonStein/Notepads/commit/f886a1cfdc618dcc51f03cf4350e680aefc346ba)

Fix looks good 👍

Comment (milotype) [reply src/Notepads/Strings/de-DE/Settings.resw:189]: sorry, not quite ... at the end of the string I also oversaw "ausschaltet" (that's wrong).
This is how the string should look like:

Opazität der Acryl-Hintergrundfarbe. Der Acryleffekt wird deaktiviert, wenn der Batteriesparmodus eingeschaltet ist oder wenn die Transparenzeffekte in den Windows-Einstellungen ausgeschaltet sind.

Sorry again ...

Comment (JasonStein) [reply src/Notepads/Strings/de-DE/Settings.resw:189]: > > [f886a1c](https://github.com/JasonStein/Notepads/commit/f886a1cfdc618dcc51f03cf4350e680aefc346ba)
> 
> Fix looks good 👍

Any comments on @milotype's suggestion?

Comment (beerisgood) [reply src/Notepads/Strings/de-DE/Settings.resw:189]: I oversaw that too but the new fix solve that

Comment (JasonStein) [reply src/Notepads/Strings/de-DE/Settings.resw:189]: So what should I use here?

Comment (milotype) [reply src/Notepads/Strings/de-DE/Settings.resw:189]: Opazität der Acryl-Hintergrundfarbe. Der Acryleffekt wird deaktiviert, wenn der Batteriesparmodus eingeschaltet ist oder wenn die Transparenzeffekte in den Windows-Einstellungen ausgeschaltet sind.

**Pull request merged**